src/main/java/org/primeframework/mvc/action/config/DefaultActionConfigurationBuilder.java

```diff
@@ -557,4 +561,62 @@ private Map<Class<?>, Object> getAdditionalConfiguration(Class<?> actionClass) {
     }
     return additionalConfiguration;
   }
+
+  private void validateNamedParameterAnnotations(Class<?> actionClass) {
+
+    var annotatedMethods = ReflectionUtils.findAllMethodsWithAnnotation(actionClass, NamedParameter.class);
+    var annotatedFields = ReflectionUtils.findAllFieldsWithAnnotation(actionClass, NamedParameter.class);
+    var allProperties = ReflectionUtils.findPropertyInfo(actionClass);
+    var allFields = ReflectionUtils.findFields(actionClass).keySet();
+    Map<String, Object> namedParameters = new HashMap<>();
+    annotatedMethods.forEach(method -> {
+      var name = method.getAnnotation(NamedParameter.class).name();
+      // find this method in the properties list, it should be there and be a valid property
+      var foundPropPair = allProperties.values()
```

Comment:
What is this particular check validating? In other words, is the method invalid, or is the name specified in the annotation invalid, or is there a collision that makes it invalid? 

I added more tests to `NamedParameterHandlerAction` to try and use these types of names.